python/cuml/test/test_metrics.py

```diff
@@ -1384,3 +1385,38 @@ def test_hinge_loss(nrows, ncols, n_info, input_type, n_classes):
                                 labels=np.unique(y))
     # compare the accuracy of the two models
     cp.testing.assert_array_almost_equal(cu_loss, cu_loss_using_sk)
+
+
+@pytest.mark.parametrize("nfeatures",
+                         [
+                             unit_param(10),
+                             unit_param(300),
+                             unit_param(3000000),
```

Comment:
Is this supposed to be `unit_param` as well? Seems to be a little large number, although I don't know how quick it is

Reply:
This test is completed under 2 seconds

Reply:
I think we should reduce the size then. CI is slower than local, and this test doesn't need to be so large